### ISSUE TITLE
Button items / actions ???

### ISSUE THREAD

**Issue by ExcelNet-Public** [bradmartin/nativescript-floatingactionbutton#95]
I am trying to use this in a Vue environment and either missing something or not getting how to use this component.  It seems all this provides is a floating button and no functionality beyond that.  

I was expecting some way when I tap on it that a list of actions / buttons / whatever would be displayed "near" the button.  Something similar to the Vue floating button sample on the Nativescript Samples page.   Does this functionality exist or can somebody provide an example how they are providing this information.  

Thanks!

**Comment by jannomeister** [comment]
yeah I agree. Like how ionic implement fab button like this one: https://ionicframework.com/docs/v3/api/components/fab/FabList/.

**Comment by bradmartin** [comment]
Ah yea, you want a 'speeddial'. I did it way back but never posted the snippet anywhere I can find :(
https://www.youtube.com/watch?v=3n8cuJZmOvw&feature=youtu.be

Basically you'd just have your main FAB with a tap event and then I animate the smaller FABs that are positioned behind the main one using a grid-layout.

**Comment by JeffersonLPLima** [comment]
> 
> 
> Ah yea, you want a 'speeddial'. I did it way back but never posted the snippet anywhere I can find :(
> https://www.youtube.com/watch?v=3n8cuJZmOvw&feature=youtu.be
> 
> Basically you'd just have your main FAB with a tap event and then I animate the smaller FABs that are positioned behind the main one using a grid-layout.

That would be amazing... I'm also searching for a FAB, or a Floacting Action Menu [like this](https://www.youtube.com/watch?v=XsdG_6i5YIk) in NS, but I didn't find nothing to help.

**Comment by euller88** [comment]
@JeffersonLPLima, @jannomeister, @ExcelNet-Public, If anyone wants the code, it was on the Nativescript Snacks website. Although it is not online anymore, its repo and all its articles are still on Github. Use those links to implement the functionality you want:

- [Brad's version](https://github.com/jlooper/nativescript-snacks/blob/master/_posts/2016-04-22-fab-button.markdown) - using Nativescript core

- [Jen Looper's version](https://github.com/jlooper/nativescript-snacks/blob/master/_posts/2016-06-05-fab-nav-angular.markdown) - this expands on what Brad does and it is using Angular.

- [Carlos Cabral's sample found on the Nativescript Marketplace](https://play.nativescript.org/?template=play-vue&id=AIQOcR&v=22) - this is a complete Vue implementation of a floating action button and speeddial independent of this plugin.

**Comment by adityamadchetti** [comment]
https://github.com/nstudio/nativescript-floatingactionbutton/issues/97#issuecomment-1062935103
